frontend/package.json

```diff
@@ -31,6 +31,7 @@
     "@mui/material": "^5.13.0",
     "@reduxjs/toolkit": "^1.9.5",
     "axios": "^1.4.0",
+    "date-fns": "^2.30.0",
```

Comment:
Probably don't need this package given that we already have the moment library installed